mitosheet/mitosheet/user/utils.py

```diff
@@ -66,18 +78,18 @@ def should_upgrade_mitosheet() -> bool:
 
     Always returns false if:
     - it is not a local installation, for obvious reasons.
-    - the package is mitosheet-private, because it is managed by an account admin
+    - if it has an admin package installed, as this is managed by an admin
 
     NOTE: if the user clicks the upgrade button in the app, then we change the upgraded 
     date to this date, so that the user doesn't get a bunch of annoying popups. This just
     pushes back when they are annoyed to upgrade!
     """
     if not is_local_deployment():
         return False
-
-    if package_name == 'mitosheet-private':
+    
+    if MITOSHEET_HELPER_PRO or MITOSHEET_HELPER_PRIVATE:
```

Comment:
Are we sure that we want to make this assumption? If we are doing this, then we want to ensure that individual pro users are not upgrading to Pro using this command, because we want them to get upgrade prompts! 

If we want to keep everyone on the same pro approach though, we could add another package called "mitosheet_no_upgrade_prompt" and have specific enterprises use them. This turns into a way of setting config params

Reply:
Yeah. I think this package versioning technique makes a lot of sense, and certainly is something we can do at some point. 

For now, I think this approach is ok. People upgrading isn't so essentially anyways, and I think reminders to do so are probably more annoying than real helpful.